test/src/test/java/org/corfudb/integration/CorfuStoreBrowserIT.java

```diff
@@ -0,0 +1,190 @@
+
+    private static int corfuStringNodePort;
+
+    private static String singleNodeEndpoint;
+
+    /* A helper method that takes host and port specification, start a single server and
+     *  returns a process. */
+    private Process runSinglePersistentServer(String host, int port) throws
+        IOException {
+        return new AbstractIT.CorfuServerRunner()
+            .setHost(host)
+            .setPort(port)
+            .setLogPath(getCorfuServerLogPath(host, port))
+            .setSingle(true)
+            .runServer();
+    }
+
+    /**
+     * Load properties for a single node corfu server before each test
+     */
+    @Before
+    public void loadProperties() {
+        corfuSingleNodeHost = PROPERTIES.getProperty("corfuSingleNodeHost");
+        corfuStringNodePort = Integer.valueOf(PROPERTIES.getProperty(
+            "corfuSingleNodePort"));
+        singleNodeEndpoint = String.format(
+            "%s:%d",
+            corfuSingleNodeHost,
+            corfuStringNodePort
+        );
+    }
+
+    /**
+     * Create a table and add data to it.  Verify that the browser tool is able
+     * to read its contents accurately.
+     * @throws IOException
+     * @throws NoSuchMethodException
+     * @throws IllegalAccessException
+     * @throws InvocationTargetException
+     */
+    @Test
+    public void browserTest() throws
+        IOException,
+        NoSuchMethodException,
+        IllegalAccessException,
+        InvocationTargetException {
+        final String namespace = "namespace";
+        final String tableName = "table";
+        Process corfuServer = runSinglePersistentServer(corfuSingleNodeHost,
+            corfuStringNodePort);
+
+        // Start a Corfu runtime
+        runtime = createRuntime(singleNodeEndpoint);
+
+        CorfuStore store = new CorfuStore(runtime);
+
+        store.openTable(
+            namespace,
+            tableName,
+            SampleSchema.Uuid.class,
+            SampleSchema.Uuid.class,
+            SampleSchema.Uuid.class,
+            TableOptions.builder().build());
+
+        final long keyUuid = 1L;
+        final long valueUuid = 3L;
+        final long metadataUuid = 5L;
+
+        SampleSchema.Uuid uuidKey = SampleSchema.Uuid.newBuilder()
+            .setMsb(keyUuid)
+            .setLsb(keyUuid)
+            .build();
+        SampleSchema.Uuid uuidVal = SampleSchema.Uuid.newBuilder()
+            .setMsb(valueUuid)
+            .setLsb(valueUuid)
+            .build();
+        SampleSchema.Uuid metadata = SampleSchema.Uuid.newBuilder()
+            .setMsb(metadataUuid)
+            .setLsb(metadataUuid)
+            .build();
+        TxBuilder tx = store.tx(namespace);
+        tx.create(tableName, uuidKey, uuidVal, metadata)
+            .update(tableName, uuidKey, uuidVal, metadata)
+            .commit();
+        runtime.shutdown();
+
+        // Invoke the browser and go through each item
+        runtime.connect();
```

Comment:
Connecting an already shutdown runtime is not a good idea. Probably we should add a check to avoid that (Not in this PR).
Just create a new runtime.